theano/sandbox/gpuarray/cudnn_helper.h

```diff
@@ -4,193 +4,110 @@
 #include <cudnn.h>
 
 #ifndef CUDNN_VERSION
```

Comment:
Probably, but the cuda backend contains lots of cudnnv1-compatible code that I removed from here.  I could do a cleanup of it too, but I decided against it.